feat/new product overview pages (#646)
hirosystems/docs: docs/ordhook/index.mdx

### Install Ordhook

Clone the [Ordhook Repo](https://github.com/hirosystems/ordhook.git):

```sh
git clone https://github.com/hirosystems/ordhook.git
```

Navigate to the root directory and run `cargo ordhook-install` to install:

```sh
cd ordhook && cargo ordhook-install
```

### Create Event Streams

<div class="subSections my-8">
  <a href="/ordhook/guides/scan-ordinal-activities">
    <div class="subSectionTitle"><h3>Scan past Ordinals activity</h3><span>→</span></div>
    <p>Fetch historical ordinals activity, whether for a range of blocks or for a particular inscription.</p>
  </a>
  <a href="/ordhook/guides/stream-ordinal-activities">
    <div class="subSectionTitle"><h3>Stream Ordinals activity</h3><span>→</span></div>
    <p>Create webhook-like triggers that listen for ordinals activity and post that activity to a local server or file.</p>
  </a>
</div>

<div class="subSections my-8 w-[50%]">
  <a href="/ordhook/guides/ordhook-as-a-service-bitcoind">

>     <div class="subSectionTitle"><h3>Run Ordhook as a Service</h3><span>→</span></div>

Reviewer: ```suggestion
    <div class="subSectionTitle"><h3>Run Ordhook as a service</h3><span>→</span></div>
```

    <p>Spin up an ongoing service that listens for future ordinals activity and posts that data to a local server.</p>
  </a>
</div>

### Related Tools

**[Ordinals API](../ordinals-api/index.mdx)**: Skip the infrastructure setup and get the ordinals data you need from our Ordinals API, which is powered by Ordhook.

**[Ordinals Explorer](https://ordinals.hiro.so)**: Explore ordinals inscriptions and BRC-20 tokens rapidly in our Ordinals Explorer.

---

> Need help building with Ordhook? Reach out to us on the `#chainhook` channel on Discord under Hiro Developer Tools section. We also host a weekly office hours on Discord every Monday at 11am ET.

Reviewer: ```suggestion
Need help building with Ordhook? Reach out to us on the `#chainhook` channel on Discord under the [Hiro Developer Tools section](https://stacks.chat/). There’s also a [weekly office hours](https://events.stacks.co/event/HD16484710) on Discord every Thursday at 11am ET.
```
